include/ensmallen_bits/cne/cne.hpp

```diff
@@ -137,16 +133,11 @@ class CNE
   //! Modify the selection percentage.
   double& SelectionPercentage() { return selectPercent; }
 
-  //! Get the final objective value.
+  //! Get the final tolerance.
```

Comment:
Pedantic style comment, do you mind to remove the "final" as I don't think the tolerance will change over time.